### PR TITLE
Add tox environment with ISOTP Linux kernel modules

### PR DESCRIPTION
**Pull request by polybassa**
How about this solution to have a custom Linux kernel module installed for some ISOTPSocket unit tests?

### CODE REVIEW

**Comment by polybassa** [comment]
Hi, this tox environment would work for me. What do you think about this solution? In addition to this pull request, I would create an issue to fix 4 unit tests on travis.

**Comment by guedou** [comment]
Looks ok to me. Does this tox environment work fine locally?

**Comment by polybassa** [comment]
How can I do a local test? Usually some other tests fail earlier, if I run tox locally.

**Comment by polybassa** [comment]
The execution of this command `tox -e py36-isotp_kernel_module --` works on my local system.

File: tox.ini [hunk 1]
```diff
@@ -31,6 +31,31 @@ passenv = {[testenv]passenv}
...
+
+passenv = PATH PWD PROGRAMFILES WINDIR SYSTEMROOT
+deps = mock
+       cryptography
+       coverage
+       python-can
+
+commands =
+  sudo -E modprobe can
+  git clone https://github.com/linux-can/can-utils.git /tmp/can-utils
```
Comment (guedou) [inline tox.ini]: Could you add `--depth=1` here?

File: tox.ini [hunk 2]
```diff
@@ -31,6 +31,31 @@ passenv = {[testenv]passenv}
...
+  bash -c "cd /tmp/can-utils; autogen.sh; ./configure; make; sudo make install"
+  git clone https://github.com/hartkopp/can-isotp.git /tmp/can-isotp
```
Comment (guedou) [inline tox.ini]: Could you add `--depth=1` here?

**Comment by guedou** [comment]
Perfect!

That is exactly what I meant. I tested it on Ubuntu 18.10 and it worked fine.

File: tox.ini (hunk 1 above)
Comment (polybassa) [reply tox.ini]: Done

File: tox.ini (hunk 2 above)
Comment (polybassa) [reply tox.ini]: Done

**Pull request merged**